src/shared/components/ChannelPreview/ChannelPreviewBase.tsx

```diff
@@ -1,37 +1,73 @@
 import React from 'react'
-import { AvatarContainer, Info, InnerContainer, OuterContainer } from './ChannelPreviewBase.style'
+import {
+  Anchor,
+  AvatarContainer,
+  Info,
+  InnerContainer,
+  NameHeader,
+  OuterContainer,
+  StyledAvatar,
+  VideoCount,
+  VideoCountContainer,
+} from './ChannelPreviewBase.style'
 import Placeholder from '../Placeholder'
+import { CSSTransition, SwitchTransition } from 'react-transition-group'
+import { transitions } from '@/shared/theme'
 
 type ChannelPreviewBaseProps = {
-  avatarNode?: React.ReactNode
-  nameNode?: React.ReactNode
+  avatarURL?: string
+  name?: string
+  videoCount?: number
+  channelHref?: string
   className?: string
   animated?: boolean
+  loading?: boolean
   onClick?: (e: React.MouseEvent<HTMLElement>) => void
 }
 
 const ChannelPreviewBase: React.FC<ChannelPreviewBaseProps> = ({
-  avatarNode,
-  nameNode,
+  avatarURL,
+  name,
+  videoCount,
+  loading = true,
+  channelHref,
   className,
   animated = false,
```

Comment:
I think we can default to `true` here

Reply:
Or even better, remove that prop altogether and assume we animate if `!loading && channelHref`